docker/docker-compose.EXAMPLE.yml

```diff
@@ -0,0 +1,81 @@
+version: "3"
+services:
+  scout:
+    image: scout:latest
+    volumes:
+      - /home/scout/vault:/scout/vault:ro
```

Comment:
I think there should be an additional volume here, binding the sample data to the container
e.g. ```- /home/scout/sampledata:/scout/sampledata:ro``` otherwise this container will not be able to access any of the files. Maybe this is why you had problems with the bam-files?

Reply:
the mounting of the sampledata folder is actually done inside the scout-import image, which builds on the scout image, and connects to the mongo-db image. I see now that I actually mounted this already in the docker-compose file, and then I mount it again in the case-handler script, I should only do it in the case-handler actually. The idea was that you could have your samples in any folder and just mount that specific folder using the case-handler script. Otherwise you'd have to hardcode the path to the folder where all the data is in, and also there's no point in mounting the entire data folder every time.

Reply:
Yes, it works fine using the scout-import image to upload cases to the database if the files are mounted properly. However, let's say a user that connects to the server in the 'scout'-container, wants to have a nice igv-view around a variant, there will be no mount to the bam-file from that container. The mounts will only exist in the 'scout-import' container while the case is imported, not in the container where the server is running. This I think is the problem.